public_html/wp-content/themes/pattern-directory/src/components/breadcrumb-monitor/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as patternStore } from '../../store';
+import { getAllCategory } from '../../store/utils';
+import { useRoute } from '../../hooks';
+import { getCategoryFromPath, getValueFromPath } from '../../utils';
+
+/**
+ * Update the breadcrumb text to the category name.
+ *
+ * @param {string} categoryName
+ */
+const setBreadcrumbText = ( categoryName ) => {
+	const breadcrumb = document.getElementById( 'breadcrumb-part' );
+	if ( breadcrumb ) {
+		const split = breadcrumb.innerText.split( ':' );
+
+		breadcrumb.innerText = `${ split[ 0 ] }: ${ categoryName }`;
```

Comment:
I don't think you can rely on this being `:` in every language, so it would be better to replace the whole "section: term" string, using a placeholder for the term, `__( 'Category: %s' )`, `__( 'Author: %s' )`.

Reply:
Replacing the whole string would also prevent this ("Author: Columns"):
<img width="281" alt="Screen Shot 2021-07-08 at 9 05 02 AM" src="https://user-images.githubusercontent.com/541093/124926496-979dfa00-dfcb-11eb-9b6e-62a7f9682108.png">

Reply:
I hadn't considered other languages. Thanks for the notice. I'll update.

In terms of the `Author` problem. That currently happens because there's a bigger problem with how we currently handle urls and how it works with`author/admin/` (it replaces it when you click on a category). I'm hesitant to handle that case in this PR because my gut is telling me we'll probably want a separate component or risk overcomplicating some of the "default" components to handle too many different cases.

Reply:
Yeah, I wasn't clear -- Author stuff can all be handled in #268. I don't think we'll need a separate component, but the header nav might need some cleanup/refactor since it's all so Category specific right now.

Reply:
Do you think there's anything left here?

Reply:
Nope, breadcrumb nav looks good 👍🏻